README.md

```diff
@@ -1,33 +1,66 @@
-Hello Microverse
+# Hello Microverse
 
-This Project is created for learning how to create repository in github and add with linker to which show error if there is have errors. and adding 3 files which is A.html, A.css and A test.md.  
+This project is created for learning that how to create github repositry and add linters file, html and css file in the project. 
 
-Built With
-Html
-Css
 
-Getting Started
-this project is for review to microverse.
+## Built With
 
+- HTML
+- CSS
+- Linters
 
-Authors
-👤 Author Samiullah
+## Live Demo (if available)
 
-GitHub: @githubhandle
-Twitter: @twitterhandle
-LinkedIn: LinkedIn
 
-🤝 Contributing
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [Optional] Let's remove the sections that are empty as they are not applicable in this project. 😃